### DISCUSSION TITLE
Are [project].xcodeproj and [project].xcworkspace generated components that should be excluded from git?

### DISCUSSION THREAD

**Discussion by luongvo** [nimblehq/ios-templates#499] [Q&A] [answered]
Hi @nimblehq/ios-chapter,

Are [project].xcodeproj and [project].xcworkspace generated components that should be excluded from git? I see [some projects](https://github.com/nimblehq/worpt-ios) pushed them along with the base source code 🤔 

If yes, should we add them to the template default [gitignore](https://github.com/nimblehq/ios-templates/blob/main/.gitignore)?
If not, should we see them from [the base structure of the ios-template](https://github.com/nimblehq/ios-templates/tree/main)?

<img width="309" alt="image" src="https://github.com/nimblehq/ios-templates/assets/16315358/940ac603-ea0c-4012-876b-a63e94daa3b1">

Thank you

**Comment by blyscuit** [comment, marked as answer]
@luongvo  No you don't exclude [project].xcodeproj / [project].xcworkspace from Git. They are not in the base structure of the iOS Template because the template uses [Tuist](https://docs.tuist.io/tutorial/get-started/) to generate these files in the setup script `make.sh`. After the first generation, Tuist is removed and xcodeproj/xcworkspace are generated

**Reply by luongvo** [reply]
@blyscuit thank you for clarifying.
